app/src/main/java/org/systers/mentorship/view/fragments/ChangePasswordFragment.kt

```diff
@@ -62,11 +64,34 @@ class ChangePasswordFragment : DialogFragment() {
         super.onResume()
 
         val passwordDialog = dialog as? AlertDialog
+        passwordDialog?.getButton(AlertDialog.BUTTON_POSITIVE)?.setEnabled(false)
+        currentPassword = changePasswordView.tilCurrentPassword?.editText?.text.toString()
+        newPassword = changePasswordView.tilNewPassword?.editText?.text.toString()
+        confirmPassword = changePasswordView.tilConfirmPassword?.editText?.text.toString()
+        changePasswordView.tilConfirmPassword?.editText?.addTextChangedListener(object : TextWatcher{
+            override fun afterTextChanged(s: Editable) {
+                if (s.length.equals(0))
+                {
+                    passwordDialog?.getButton(AlertDialog.BUTTON_POSITIVE)?.setEnabled(false)                }
+            }
+
+            override fun beforeTextChanged(s: CharSequence, start: Int,
+                                           count: Int, after: Int) {
+            }
+
+            override fun onTextChanged(s: CharSequence, start: Int,
```

Comment:
@Priyansh-Kedia instead of checking inside `onTextChanged` it will be better to implement this inside `afterTextChanged` only as we do not need the exact changes. Also please follow CamelCase while naming your variables